bangazonapi/views/lineitem.py

```diff
@@ -76,7 +76,9 @@ def destroy(self, request, pk=None):
         """
         try:
             customer = Customer.objects.get(user=request.auth.user)
-            order_product = OrderProduct.objects.get(pk=pk, order__customer=customer)
+            #order_product = OrderProduct.objects.get(pk=pk, order__customer=customer)
```

Comment:
Since this line doesn't need to be here it should be deleted instead of commented out